### PR TITLE
style(typings): export Tab definitions

### PR DESCRIPTION
**Pull request by pierre-H**

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -156,6 +156,8 @@ export { default as Sidebar, SidebarProps } from './dist/commonjs/modules/Sideba
 export { default as SidebarPushable, SidebarPushableProps } from './dist/commonjs/modules/Sidebar/SidebarPushable';
 export { default as SidebarPusher, SidebarPusherProps } from './dist/commonjs/modules/Sidebar/SidebarPusher';
 
+export { default as Tab, TabProps} from './dist/commonjs/modules/Tab';
+
```
Comment (layershifter) [inline index.d.ts]: The `Tab` has also the `TabPane` subcomponent, let's export it, too

**Review by layershifter — CHANGES_REQUESTED** [review]

File: index.d.ts (hunk 1 above)
Comment (pierre-H) [reply index.d.ts]: @layershifter You're right ! It's done.

**Review by layershifter — APPROVED** [review]
@pierre-H Thanks for update, LGTM

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.71.0`.

**Pull request merged**